core/domain/topic_domain.py

```diff
@@ -441,6 +450,11 @@ def validate(self):
         if self.thumbnail_filename and self.thumbnail_bg_color is None:
             raise utils.ValidationError(
                 'Subtopic thumbnail background color is not specified.')
+        if self.thumbnail_filename is not None and (
+                self.thumbnail_size_in_bytes is 0):
```

Comment:
Technically this should be `==` not `is`.

Reply:
Facing the lint error, when using `!=` instead of `is not None`
![image](https://user-images.githubusercontent.com/36214903/124857133-602e4e00-dfc9-11eb-99a9-e4ba5b3e3619.png)

Reply:
Done.